pkg/daemon/daemon.go

```diff
@@ -692,19 +691,15 @@ func (dn *Daemon) runKubeletHealthzMonitor(stopCh <-chan struct{}, exitCh chan<-
 		case <-stopCh:
 			return
 		case <-time.After(kubeletHealthzPollingInterval):
-			if err := dn.getHealth(); err != nil {
-				glog.Warningf("Failed kubelet health check: %v", err)
+			err := dn.getHealth()
+			if err != nil {
 				failureCount++
-				KubeletHealthState.WithLabelValues(err.Error()).Set(float64(failureCount))
-				if failureCount >= kubeletHealthzFailureThreshold {
```

Comment:
do you not want to keep the kubeletHealthzFailureThreshold just to gate when an error is sent to the exitCh?

Reply:
we were logging it before on old line 696 and i don't see any side effect of just sending it to the channel for logging.  that way we don't log it twice when we are over the threshold.

Reply:
fair enough :+1: